### ISSUE TITLE
[BUG] "Subject Average" section in the assessments page not showing

### ISSUE THREAD

**Issue by NNIDNHU** [BetterSEQTA/BetterSEQTA-Plus#190]
**Describe the bug**
same as title

**To Reproduce**
open asessments page and look at asessments

**Expected behaviuor**
there is an option for subject averages 

**Screenshots**
<img width="164" alt="1" src="https://github.com/user-attachments/assets/ec686937-40c5-40f5-8419-fcb47febb196">

**Desktop (please complete the following information):**
 - OS: Windows 10 (19045.5131)

**Comment by ar-cyber** [comment]
Can recreate.

**Comment by NNIDNHU** [comment]
ah ok, should probably have been better documented

**Comment by NNIDNHU** [comment]
still does not show up

**Comment by NNIDNHU** [comment]
i still cannot find it

**Comment by SethBurkart123** [comment]
> ah ok, should probably have been better documented

yeah it was kinda an accident on my part. I forgot to make it auto enable. It should be this switch

<img width="396" alt="image" src="https://github.com/user-attachments/assets/0c0b5d2a-c696-4e6a-912d-8d3fe394f2e8">

**Comment by NNIDNHU** [comment]
> > ah ok, should probably have been better documented
> 
> yeah it was kinda an accident on my part. I forgot to make it auto enable. It should be this switch
> 
> <img alt="image" width="396" src="https://private-user-images.githubusercontent.com/108050083/391768485-0c0b5d2a-c696-4e6a-912d-8d3fe394f2e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMxODkwMzgsIm5iZiI6MTczMzE4ODczOCwicGF0aCI6Ii8xMDgwNTAwODMvMzkxNzY4NDg1LTBjMGI1ZDJhLWM2OTYtNGU2YS05MTJkLThkM2ZlMzk0ZjJlOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwM1QwMTE4NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NjMxOTBhZmE4OTMwZDFhZTJkMzkzNDBjY2YzOWY0Mjg0ODc1MjdkMWEzYTc1MjI3NGJkMjkxMDQyZjQyN2EyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6tjq2PNLA4E0EcSbVzOjooNwxW1rCaZJ9x_7IcMDrA0">

I enabled it, however for some reason it still does not show until i restart chrome and then it only shows for like half of my subjects.

Example of shown subject average:
<img width="341" alt="Shown average" src="https://github.com/user-attachments/assets/cd0834a9-1593-4ef7-a8e0-a3a6b33e3a24">

Example of missing subject average:
<img width="335" alt="Missing average" src="https://github.com/user-attachments/assets/36214ff4-fce7-4a98-8bb4-73ce736e3b3a">

**Comment by ar-cyber** [comment]
Known bug: Grades without percentages do not have the averages.

**Comment by SethBurkart123** [comment]
Yeah, if it runs into any issues it doesn't show. So I'll just have a look into making it work in more cases

**Comment by ar-cyber** [comment]
fixes #191

**Comment by ar-cyber** [comment]
Someone please close this.
